### ISSUE TITLE
The player should be able to unlock (buy?) new characters

### ISSUE THREAD

**Issue by Jontpan** [AxlLind/2D-platformer#95]
I was also thinking that we could have fun names and descriptions for the different characters.

**Comment by Jontpan** [comment]
Maybe not necessary for google play release? Should we remove it from the milestone?

**Comment by AxlLind** [comment]
Yes, I think we could work on this after. Not a priority right now.
